sdk/core/azure-core/src/main/java/com/azure/core/http/policy/ExponentialBackoffOptions.java

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.http.policy;
+
+import java.time.Duration;
+
+/**
+ * The configuration for exponential backoff.
+ */
+public class ExponentialBackoffOptions {
```

Comment:
can you please add some tests for new options? they'd also come handly after i add configuration properties :)